.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install -U codecov pytest pytest-cov pyyaml trollsift posttroll inotify pyinotify paramiko scp watchdog
+          pip install -U codecov pytest pytest-cov pytest-bdd pytest-reraise pyyaml trollsift posttroll inotify pyinotify paramiko scp watchdog pytroll-collectors
```

Comment:
`pytroll-collectors` needs to be added as an optional dependency in `setup.py`, and `pytest-bdd` and `pytest-reraise` as test requirements.

Reply:
thanks, adding

Reply:
hmm, the documentation of setuptools says `tests_require` is deprecated (since `python setup.py test` is deprecated)... I'm not sure how to add the tests requirements now.